templates/clients.html

```diff
@@ -383,7 +404,7 @@ <h4 class="modal-title">Remove</h4>
             }
 
             const data = {"id": client_id, "name": name, "email": email, "allocated_ips": allocated_ips,
-                "allowed_ips": allowed_ips, "use_server_dns": use_server_dns, "enabled": enabled};
+                "allowed_ips": allowed_ips, "extra_allowed_ips": extra_allowed_ips, "use_server_dns": use_server_dns, "enabled": enabled};
```

Comment:
You have `extra_allowed_ips` data in this `submitEditClient()` function but you missed it in [submitNewClient()](https://github.com/ngoduykhanh/wireguard-ui/blob/e74a3a808f5ab2c2cb68662742885c85d97126bf/templates/base.html#L307-L308) function. So the extra allowed IP addresses only work when we edit a client, not when adding.

Reply:
Also, I have one comment on the generated client's json files in the DB. If we don't specify **Extra Allowed IPs**, its json data has the following value
```
{
	"id": "c7q6bh7m20rnopklh1r0",
         .....
	"extra_allowed_ips": [
		""
	],
	 ...
}%
```
Can we set `extra_allowed_ips` to an empty list instead of a list that has one empty string item?

Reply:
I will check this out tonight and update PR. Thank you. I think I may have missed a push, but i'll make these modifications and re-test.

Reply:
Reworked commit, defaulted to empty list.

Reply:
LGTM now. Thanks @brittondodd